packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -335,3 +338,51 @@ void _writeGeneratedFlutterConfig(
   }
   writeGeneratedCmakeConfig(Cache.flutterRoot!, windowsProject, environment);
 }
+
+// Works around the Visual Studio CMake bug described in
+// https://github.com/flutter/flutter/issues/97086
```

Comment:
should we also link https://developercommunity.visualstudio.com/t/CMake-vcxproj-regression-in-1710-when/1670541 ?

Reply:
Since the Flutter issue already links to the VS issue, I figured it was better to keep it simple here.